internal/privi/repo.go

```diff
@@ -5,40 +5,54 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"strconv"
+	"strings"
 
 	"github.com/bluesky-social/indigo/atproto/atdata"
-	_ "github.com/mattn/go-sqlite3"
+	"github.com/ipfs/go-cid"
+	"github.com/multiformats/go-multihash"
 )
 
 // Persist private data within repos that mirror public repos.
-// A repo implements two methods: putRecord and getRecord.
+// A repo currently implements four basic methods: putRecord, getRecord, uploadBlob, getBlob, meant to implement the same interfaces as ATproto
+// In the future, we should use those lexicons directly
```

Comment:
i don't think we'll be able to do this and imo that's fine. seems like there are some fundamental differences once permissions are involved